sky/clouds/service_catalog/data_fetchers/fetch_azure.py

```diff
@@ -244,11 +244,79 @@ def get_additional_columns(row):
         axis='columns',
     )
 
+    def create_gpu_map(df):
+        # Map of Azure's machine with GPU to their corresponding memory
+        # Result is hard-coded since Azure's API to not return such info
+        # may be outdated so need to be maintained
```

Comment:
Could you add a reference link on how these information are found? also, how did we make sure we cover all the instance types?

Reply:
I looked through the azure documentation to ensure that all instance type have been included and also ran the script on --all-regions and the result looked fine to me.
However, I think the approach you mentioned below makes more sense in which we map instance type -> gpu name  then from gpu name -> gpu memory. There is already a mapping from instance type -> gpu name in the script. Assuming this mapping is complete, we can easily map the gpu name to their corresponding memory.